rfb/rfbclient.h

```diff
@@ -295,13 +297,16 @@ typedef struct _rfbClient {
 	uint8_t tightPrevRow[2048*3*sizeof(uint16_t)];
 
 #ifdef LIBVNCSERVER_HAVE_LIBJPEG
-	/** JPEG decoder state. */
+	/** JPEG decoder state (obsolete-- do not use). */
 	rfbBool jpegError;
 
 	struct jpeg_source_mgr* jpegSrcManager;
 	void* jpegBufferPtr;
 	size_t jpegBufferLen;
 
+	/** JPEG decoder state. */
+	tjhandle tjhnd;
```

Comment:
i meant: to the end of the struct. the way it is now, other struct members after this one are offset by sizeof(tjhandle).